src/Projection/PdoEventStoreReadModelProjector.php

```diff
@@ -595,8 +596,11 @@ private function createProjection(): void
 EOT;
 
         $statement = $this->connection->prepare($sql);
-        // we ignore any occuring error here (duplicate projection)
-        $statement->execute([$this->name, $this->status->getValue()]);
+        try {
+            $statement->execute([$this->name, $this->status->getValue()]);
+        } catch (PDOException $e) {
```

Comment:
The comment suggested to me that it was supposed to fall through. I assumed it was set up this way to save a SELECT to see if it exists.

However, I think I need to add a check so that only error 1062 passes through.